scripts/evaluate/evaluate.py

```
@@ -172,40 +237,71 @@ def run_benchmark(args: argparse.Namespace) -> None:
-            all_max_tokens[subset] = mt
+    dataset_spec = args.dataset
+    is_speedbench = dataset_spec.startswith("speedbench/")
+
+    if is_speedbench:
+        if not getattr(args, "speedbench_data_dir", None):
+            logger.error(
+                "--speedbench-data-dir is required for speedbench/ datasets.\n"
+                "Prepare the data first:\n"
+                "  curl -LsSf %s | python3 - --output_dir ./speedbench_data\n"
+                "Then add: --speedbench-data-dir ./speedbench_data",
+                _SPEEDBENCH_PREPARE_URL,
```

Ruff (0.15.20)

[error] Undefined name `_SPEEDBENCH_PREPARE_URL`

(F821)